tests/Scale/AbstractStampedeProtectorTestCase.php

```diff
@@ -197,6 +197,57 @@ public function testStampedeGetMultiStampede(): void
         }
     }
 
+    public function testStampedeGetExistingAndProtected(): void
+    {
+        /*
+         * this test is for a rare occurrence where one request protects a key
+         * and resolves the value very fast but the other request reads both
+         * the stampede and the value but still waits and it shouldn't
+         */
+
+        if (!$this->isForkable()) {
+            $this->markTestSkipped("Can't test stampede without forking");
+        }
+
+        $this->adapterKeyValueStore->set('key', 'value');
+        $this->adapterKeyValueStore->set('key.stampede', '');
+
+        $pid = pcntl_fork();
```

Comment:
I don't think we need the forking here; just the `set`s above, and the `getMulti` + `assertEquals` further down.

Reply:
Nevermind. I see now you already simplified it. Looking forward to the release. ❤️

Reply:
Just waiting for [the tests](https://github.com/matthiasmullie/scrapbook/actions/runs/7472909494/job/20336013470) to pass; then I'll push a new tag! Thanks man!